builder/virtualbox/ovf/config_test.go

```diff
@@ -76,17 +76,6 @@ func TestNewConfig_sourcePath(t *testing.T) {
 		t.Fatalf("Nonexistant file should throw a validation error!")
 	}
 
-	// Bad
```

Comment:
I see that you've added support for ftp.  Given that this support is an enhancement rather than a bugfix, and unrelated to the change to DownloadableURL, can we move that into a separate PR so I can review it separately?  I'm hesitant to even ask you this because in the near-to-middle-term we have a goal of using https://github.com/hashicorp/go-getter for downloads which I believe already solves that problem.  I don't want to make you replicate work we might be tossing out soon anyway.

Reply:
Yeah, sure. I can do that. I added ftp because it was asked on another issue and it wasn't too much work to add.